pages/in-vitro-systems/[id].js

```diff
@@ -130,6 +122,22 @@ export default function InVitroSystem({
                     </DataItemValue>
                   </>
                 )}
+                {truthyOrZero(inVitroSystem.growth_medium) && (
```

Comment:
`truthyOrZero` is for numeric values, but `growth_medium` is a string. You can simply test for its existence.

Reply:
done